applications/connecting_applications_to_services/exposing-binding-data-from-a-service.adoc

```diff
@@ -0,0 +1,21 @@
+[id="exposing-binding-data-from-a-service"]
+= Exposing binding data from a service
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: exposing-binding-data-from-a-service
+
+toc::[]
+
+Application developers need access to backing services to build and connect workloads. Connecting workloads to backing services is always a challenge as each service provider suggests a different way to access their secrets and consume them in a workload.
+
+{servicebinding-title} enables application developers to easily bind workloads together with operator-managed backing services, without any manual procedures to configure the binding connection. For the {servicebinding-title} to provide the binding data, as an Operator provider or user who creates backing services, you must expose the binding data to be automatically detected by the {servicebinding-title}. Then, the {servicebinding-title} automatically collects the binding data from the backing service and shares it with a workload providing a consistent and predictable experience.
+
+include::modules/sbo-methods-of-exposing-binding-data.adoc[leveloffset=+1]
+
+include::modules/sbo-categories-of-exposable-binding-data.adoc[leveloffset=+1]
+
+== Additional resources
```

Comment:
In the "Additional Resources" section, you might consider removing the "For more information, see..." introductory phrases from each bullet point and just providing the link names, which in this case are self-explanatory about what information they are pointing to. It's not strictly necessary, of course, but I like to keep things as minimal as possible.